js/source/worker.js

```diff
@@ -14,7 +14,7 @@ var rewind = require('geojson-rewind');
 var GeoJSONWrapper = require('./geojson_wrapper');
 var vtpbf = require('vt-pbf');
 
-module.exports = function(self) {
+module.exports = function MapboxGlWorker(self) {
```

Comment:
It should be fixed, correct. However, I'm on `webworkify-webpack@1.1.1` and `mapbox-gl@0.20.1` using the recommended webpack.config.js setup. This produced the error mentioned in the pull request. Adding a name to the function fixed the issue for me.